modules/docs/arrow-docs/docs/docs/arrow/typeclasses/divide/README.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: docs
+title: Divide
+permalink: /docs/arrow/typeclasses/divide/
+redirect_from:
+  - /docs/typeclasses/divide
+---
+
+## Divide
+
+{:.beginner}
+beginner
+
+`Divide` is a typeclass that models the divide part of divide and conquer.
+`Divide` basically states: Given a `Kind<F, A>` and a `Kind<F, B>` and a way to turn `C` into an `A` and `B` it provides you a `Kind<F, C>`
```

Comment:
Same as above, mention a coproduct or a tuple or something that helps with the association, else it's easy to miss.